sideboard/tests/test_server.py

```diff
@@ -19,10 +18,15 @@
 
 import sideboard.websockets
 from sideboard.lib import log, config, subscribes, notifies, services, cached_property, WebSocket
-from sideboard.tests import service_patcher, config_patcher
+from sideboard.tests import service_patcher, config_patcher, get_available_port
 from sideboard.tests.test_sa import Session
 
 
+available_port = get_available_port()
+config['cherrypy']['server.socket_port'] = available_port
```

Comment:
I'm not sure of all the places it's being used, but I didn't want another developer in the future to try using it and wonder why it's not working. I wanted to make sure that `config['cherrypy']['server.socket_port']` always contains the value that we'd expect (principle of least astonishment).